### ISSUE TITLE
About training data

### ISSUE THREAD

**Issue by JL6666JL** [lose4578/SAM-DiffSR#11]
I downloaded mask data from Google Driver and I found that there are a total of 3550 masks. Did you put DIV2K_valid_HR into the training set (800 + 100 + 2650 = 3550)? If that's the case, how to test it on DIV2K, because DIV2K_test_HR seems to be unobtainable.
